feat(example): add nx example (#390)
vercel/examples: solutions/nx-monorepo/README.md

---
name: Monorepo with Nx
slug: monorepo-nx
description: Learn to implement a monorepo with a single Next.js site using Nx.
framework: Next.js
useCase: Documentation
css: None
deployUrl: https://vercel.com/new/git/external?repository-url=https://github.com/vercel/examples/tree/main/solutions/nx-monorepo&project-name=nx-monorepo&repository-name=nx-monorepo&output-directory=dist%2Fapps%2Fapp%2F.next&build-command=npx%20nx%20build%20app%20--prod&ignore-command=npx%20nx-ignore%20app%20
demoUrl: https://solutions-nx-monorepo.vercel.sh
---

# Nx Monorepo

> This is a monorepo example with a single Next.js site ([./app](./app)).
> 
> This example was created using [Nx](https://nx.dev).

Reviewer: ```suggestion
This is a monorepo example using [Nx](https://nx.dev) and a single Next.js site in [./apps/app](./apps/app).
```
